src/capitalsFirst.js

```diff
@@ -21,7 +21,20 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  let splitedArray = str.split(' ');
+  let lowers = [];
```

Comment:
suggestion: lowerCaseWords.

Reply:
done, thanks to ide its not so hard to change all names ))